react/ProductSummary.js

```diff
@@ -27,85 +26,98 @@ class ProductSummary extends Component {
-                      showBadge ? (
-                        <DiscountBadge
-                          listPrice={product.listPrice}
-                          sellingPrice={product.sellingPrice}
-                          label={badgeText}>
-                          <img className="vtex-product-summary__image" alt={product.name} src={product.imageUrl} />
-                        </DiscountBadge>
-                      ) : (
-                        <img className="vtex-product-summary__image" alt={product.name} src={product.imageUrl} />
-                      )
-                    }
-                  </div>
-                  <div className="vtex-product-summary__name-container pv5 f4 gray db tc">
-                    <ProductName
-                      name={product.name}
-                      skuName={product.skuName}
-                      brandName={product.brandName}
-                      referenceCode={product.referenceCode} />
-                  </div>
-                  <div className="vtex-price-container pv1">
-                    <Price
-                      listPrice={product.listPrice}
-                      sellingPrice={product.sellingPrice}
-                      installments={product.installments}
-                      installmentPrice={product.installmentPrice}
-                      showListPrice={showListPrice}
-                      showLabels={showLabels}
-                      showInstallments={showInstallments} />
-                  </div>
-                </div>
-                <div className="pv2">
-                  <div className={`${(!showButtonOnHover || (showButtonOnHover && this.state.isHovering)) ? 'db' : 'dn'}`}>
-                    {!hideBuyButton && (
-                    // TODO: Use the buy button component
-                      <div className="vtex-product-summary__buy-button center">
-                        <Button block primary onClick={event => event.stopPropagation()}>BUY</Button>
-                      </div>
-                    )}
-                  </div>
+            <div className="vtex-product-summary__name-container pv5 f4 gray db tc">
+              <ProductName
+                name={product.name}
+                skuName={product.skuName}
+                brandName={product.brandName}
+                referenceCode={product.referenceCode}
+              />
+            </div>
+            <div className="vtex-price-container pv1">
+              <Price
+                listPrice={product.listPrice}
+                sellingPrice={product.sellingPrice}
+                installments={product.installments}
+                installmentPrice={product.installmentPrice}
+                showListPrice={showListPrice}
+                showLabels={showLabels}
+                showInstallments={showInstallments}
+              />
+            </div>
+          </div>
+          <div className="pv2">
+            <div
+              className={`${
+                !showButtonOnHover ||
```

Comment:
This expression could be simplified to `(!showButtonOnHover || this.state.hovering)`